### DISCUSSION TITLE
Автообновление zapret-ip-exclude.txt

### DISCUSSION THREAD

**Discussion by numbereleven-a** [bol-van/zapret#351] [General]
del

**Comment by numbereleven-a** [comment]


**Reply by bol-van** [reply]
все файлы  zapret-ip* формируются скриптами
они обновляются ночью по крону
подтягиваются внешние листы, ресолвятся юзерские
ип адреса могут меняться. надо пере ресолвливать периодически
все юзерское вписывется в zapret-hosts-user*. txt

**Reply by bol-van** [reply]
после внесения изменений в юзер листы
само оно не начнет ресолвиться, а демоны не подхватят изменение сразу
надо прогонять  get_config.sh
если используется  hostlist, то по ip работают лишь exclude  и  ipban
exclude работает и по ip, и по  hostlist

**Reply by neduus** [reply]
> сейчас заметил что перестали некоторые сайты открываться которые до этого работали тот же рутрекер, совпадение или нети

похоже, что рутрекер перешел на quic

**Reply by bol-van** [reply]
cloudflare давно поддерживает куик, а рутакер давно за cloudflare

**Reply by neduus** [reply]
Не заметил в какой момент это произошло, но раньше не открывался без обхода и вдруг на днях на одном провайдере стал открываться, думал, что сняли блокировку, смотрю, а он quic.

**Reply by neduus** [reply]
В любом случае мне понравилась схема, ютуб для всей домашней сети через nfqws, дополнительно для пк и смарта проход посредством nfqws до cf warp. Для тех, кто сами закрываются, типа intel и прочих, отдельные решения.
Способов обхода море, но данный софт, по моему, самый элегантный. Снимаю шляпу.
